scripts/asm-installer/cloudbuild-master.yaml

```diff
@@ -144,6 +144,24 @@ steps:
   - '_CI_ASM_KPT_BRANCH=${_ASM_PKG_BRANCH}'
   timeout: 600s
 
+- name: 'gcr.io/$PROJECT_ID/${_IMAGE_NAME}'
```

Comment:
Can you update the lint rule to also look at asm/vm/asm_vm? Also, since you're now creating Cloud resources, you need to have some sort of logic to clean up ones that leak.

Reply:
Done.

Reply:
> Also, since you're now creating Cloud resources, you need to have some sort of logic to clean up ones that leak.

The test script cleans up leaked objects. Is there somewhere else that also cleans up stuff?

Reply:
You're creating service accounts and instance templates in the test and the script might error and close before it reaches the end.

Reply:
Sorry, I misread your comment at first. It's in the setup_longterm_cluster script. That way things that leak will get cleaned up eventually on the next test run.

Reply:
Added cleanup all vm instance template and cleanup all workload service accounts to the build step.